torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19821,6 +19835,16 @@ def __init__(
         "_refs.pow",
         torch_opinfo_name="pow",
     ),
+    ElementwiseBinaryPythonRefInfo(
+        "_refs.remainder",
+        torch_opinfo_name="remainder",
+        skips=(
+            DecorateInfo(unittest.skip("Skipped!"), 'TestCommon', 'test_python_ref',
```

Comment:
What happens in these two tests?

Reply:
The `ref_result` has nans whereas the `torch_result` doesn't. This is the same as in @Aidyn-A's fmod ref where these two tests are also skipped. Finding the root cause is a bit involved as I can't reproduce the nans when executing the inputs in isolation: https://gist.github.com/syed-ahmed/c1e4c5d638ce934ca18a4ce3f4844c8e.

Reply:
That's extremely interesting -- we should try to understand that better.

So the same input in the test produces a NaN in the output, but when run outside the tests the NaN isn't produced? 

That's weird... is it somehow because of this context?

https://github.com/pytorch/pytorch/blob/14b0e9e75f528686185714fdece2e060822b3537/test/test_ops.py#L479

That should remap `torch.foo` calls to `refs.foo` calls, but if the references don't call operations in the `torch` namespace it should have no effect

Reply:
Let me be a bit clearer. ref_result is as follows:
```
(Pdb) ref_result
tensor([ 1.3906e+00,  2.3750e+00, -4.8750e+00,  7.5000e-01,  2.4844e+00,
        -1.8750e-01,  6.8750e-01,  1.9844e+00, -2.4531e+00,  5.0000e-01,
        -2.3125e+00,  7.0312e-01,  6.3750e+00, -3.5781e+00,  3.9375e+00,
        -5.4375e+00, -7.5938e+00, -1.5625e+00,  3.4375e+00,         nan,
         0.0000e+00,  3.1250e+00, -2.6875e+00, -3.1875e+00,  8.7500e-01,
...
...
```
And torch_result:
```
tensor([ 1.3906e+00,  2.3750e+00, -4.8750e+00,  7.5000e-01,  2.4844e+00,
        -1.8750e-01,  6.8750e-01,  1.9844e+00, -2.4531e+00,  5.0000e-01,
        -2.3125e+00,  7.0312e-01,  6.3750e+00, -3.5781e+00,  3.9375e+00,
        -5.4375e+00, -7.5938e+00, -1.5625e+00,  3.4375e+00,  0.0000e+00,
         0.0000e+00,  3.1250e+00, -2.6875e+00, -3.1875e+00,  8.7500e-01,
...
...
```
So torch_result has 0 where ref_result has nan. With the attached gist above, I have nans for both. So I'm confused...

Reply:
Ok sounds like @Aidyn-A has a fix: https://github.com/pytorch/pytorch/pull/78837

Reply:
@Aidyn-A mentions at https://github.com/pytorch/pytorch/issues/78389:

> The reason why this happens is the use of a.copy_(b) while calling the reference.
> The copy_ method rounds down small numbers like 1.1755e-38 to exact zero for large tensors.

Reply:
> Ok sounds like @Aidyn-A has a fix: #78837

Yeah. The test in #78837 will work around this `1.1755e-38` to `0` conversion. However, I want to understand why `a.copy_(b)` does that. :thinking:

Reply:
Is this a vectorized CPU copy issue? We could test that hypothesis by disabling the vectorized CPU copy path. Let's create an independent issue with a clear reproduction for inplace copy not correctly copying values.

Reply:
@ptrblck and I are working on it

Reply:
@mruberry Leaving these xfails as is for now until we come to a conclusion on how to handle the fmod bf16 issues.